docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,163 @@
+---
+id: contributors-guide-to-open-sauced
+title: 'Contributors Guide to OpenSauced'
+sidebar_label: Contributors Guide to OpenSauced
+keywords:
+  - contributors
+  - guides
+---
+
+## Introduction to OpenSauced for Contributors
+
+OpenSauced is a platform dedicated to empowering you, the contributor, as you grow in your open source journey. We offer a GitHub-powered dashboard for tracking your contributions and gaining insights into your favorite open source projects. You can also customize your Insights and Lists Pages, connect with other contributors, and use our Highlights feature to display your contributions to the world.
+
+## Getting Started
+
+To get started, you will need to create an account on OpenSauced. You can do this by visiting the [app.opensauced.pizza](https://app.opensauced.pizza) and clicking the "Connect with GitHub" button. You will be prompted to sign in with your GitHub account. Once you have signed in, you will be redirected to the OpenSauced dashboard.
+
+## Highlights: Connecting and Recognizing Contributions
```

Comment:
Actually, the second section should be about completing your profile.  I just recorded a 5 minute video that might make this all clearer. Let me know if you can't view it. I think I got the permissions right. 

https://www.loom.com/share/a2f4d2a1a8ff4c8c96c47c4fb50a0435?sid=a4e5cb87-00d8-4044-9956-c48408c192d3

Reply:
Hi @BekahHW, I managed to view the video. So, should I write the steps there or put the link to the video?

Reply:
@CBID2 accessibility-wise, I think having written instructions is equeally important as we want to make our docs as accessible as possible for everyone. We need to think about people who are blind or have slow internet connections.

Reply:
Totally agree with @adiati98. Also, it's much easier to update text than it is video. We're going to be making some naming changes at OpenSauced in the near future, so any video that has the old naming would need to be re-recorded.